crates/sui-indexer-alt-framework/src/pipeline/concurrent/pruner.rs

```diff
@@ -473,10 +475,7 @@ mod tests {
     }
 
     #[derive(FieldCount)]
-    pub struct StoredData {
-        pub cp_sequence_number: u64,
-        pub tx_sequence_numbers: Vec<u64>,
-    }
+    pub struct StoredData {}
```

Comment:
nit: no need to add curlies for empty structs.

```suggestion
    pub struct StoredData;
```